modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,134 @@ It also includes enhancements to the Eventing and Search services.
+
+| https://issues.couchbase.com/browse/MB-32042[MB-32042^]
+| *Summary*: Fixed an issue where encrypted XDCR was incorrectly using port 8091.
+
+| https://issues.couchbase.com/browse/MB-31857[MB-31857^]
+| *Summary*: An error was thrown when DNS alternate addresses were used. This has been fixed so utilities return the hostname even if they can't find the port numbers.
+
+| https://issues.couchbase.com/browse/MB-31764[MB-31764^]
+| *Summary*: The unit of interval for XmemSelfMonitorInterval was incorrectly set to millisecond instead of second, causing an increase in the frequency of Xmem self monitor and the spurious Xmem is stuck error. This has been fixed.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v601-kv-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-32181[MB-32181^]
+| *Summary*: If one or more Ephemeral buckets were present, then nonIO background tasks could be incorrectly scheduled, potentially resulting in DCP connection instability (premature disconnection). This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-32136[MB-32136^]
+| *Summary*: When data corruption is detected on a data service node, relevant information that can help troubleshoot is now being logged.
+
+| https://issues.couchbase.com/browse/MB-32135[MB-32135^]
+| *Summary*: In some cases, the memory allocation failures were not handled causing "Write Commit Failure" errors or corruption in vBucket files.
+|===
+
+==== Eventing Service
+
+[#table_fixedissues_v601-eventing,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31923[MB-31923^]
+| *Summary*: Following the failover of a vBucket, the Eventing service restarted streams with seqno from the latest failover log entry instead of using the entry where seqnoInFailoverLog <= lastProcessedSeqno. This resulted in missed events.
+
+| https://issues.couchbase.com/browse/MB-31922[MB-31922^]
+| *Summary*: Fixed an issue where some events were missing after a source bucket flush operation.
+|===
+
+==== Index Service
+
+[#table_fixedissues_v601-gsi-views,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31989[MB-31989^]
+| *Summary*: Improved the DCP rollback handling mechanism. Instead of ignoring the disk snapshots before rolling back, the index service will now exhaust trying DCP stream requests with all the disk snapshots before rolling back to 0.
+
+| https://issues.couchbase.com/browse/MB-31744[MB-31744^]
+| *Summary*: An error is seen during a SUM aggregate pushdown when the entry value is greater than MaxInt64.
+
+| https://issues.couchbase.com/browse/MB-31724[MB-31724^]
+| *Summary*: Fixed an issue to enable creating secondary indexes on binary documents on meta.id(), meta.cas(), and meta.expiration().
+
+| https://issues.couchbase.com/browse/MB-31587[MB-31587^]
+| *Summary*: The index service crashed if the data bucket was flushed while the initial index was being built.
```

Comment:
The index service could crash under some conditions if the data bucket was flushed while the index was being built

Reply:
Thanks.